docs/reference/ml/df-analytics/apis/get-inference-trained-model-metadata.asciidoc

```diff
@@ -0,0 +1,736 @@
+(string)
+include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=model-id]
+
+`total_feature_importance`:::
+(array)
+An array of the total feature importance for each training feature used from
+the training data set.
++
+.Properties of total feature importance
+[%collapsible%open]
+=====
+
+`feature_name`:::
+(string)
+include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=inference-metadata-feature-importance-feature-name]
+
+`importance`:::
+(object)
+A collection of feature importance statistics related to the training data set for this particular feature.
++
+.Properties of feature importance
+[%collapsible%open]
+======
+`mean_magnitude`:::
+(double)
+include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=inference-metadata-feature-importance-magnitude]
+
+`max`:::
+(int)
+include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=inference-metadata-feature-importance-max]
+
+`min`:::
+(int)
+include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=inference-metadata-feature-importance-min]
+
+======
+
+`classes`:::
+(array)
+If the trained model is a classification model, feature importance statistics are gathered
+per target class value.
++
+.Properties of class feature importance
+[%collapsible%open]
+
+======
+
+`class_name`:::
+(string)
+The target class value. Could be a string, boolean, or number.
+
+`importance`:::
+(object)
+A collection of feature importance statistics related to the training data set for this particular feature.
++
+.Properties of feature importance
+[%collapsible%open]
+=======
+`mean_magnitude`:::
+(double)
+include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=inference-metadata-feature-importance-magnitude]
+
+`max`:::
+(int)
+include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=inference-metadata-feature-importance-max]
+
+`min`:::
+(int)
+include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=inference-metadata-feature-importance-min]
+
+=======
+
+======
+
+=====
+
+====
+
+[[ml-get-inference-metadata-response-codes]]
+== {api-response-codes-title}
+
+`404` (Missing resources)::
+  If `allow_no_match` is `false`, this code indicates that there are no
+  resources that match the request or only partial matches for the request.
+
+[[ml-get-inference-metadata-example]]
+== {api-examples-title}
+
+The following example gets training metadata for all the trained models:
+
+[source,console]
+--------------------------------------------------
+GET _ml/inference/_metadata
+--------------------------------------------------
+// TEST[skip:TBD]
+
+
+The API returns the following results:
+
+[source,console-result]
+----
+{
+  "count" : 2,
+  "trained_models_metadata" : [
+    {
+      "model_id" : "avg_price_prediction-1599149443166",
+      "total_feature_importance" : [
+        {
+          "feature_name" : "Origin",
+          "importance" : {
+            "mean_magnitude" : 25.862683737654795,
+            "min" : -188.93284143727874,
+            "max" : 162.8783518094679
+          }
+        },
+        {
+          "feature_name" : "FlightTimeMin",
```

Comment:
This is a very long example, so it's worth considering putting "..." in a couple of places where the info is redundant.